src/IssueViz.Security.UnitTests/OpenInIDE/Http/HttpListenerFactoryTests.cs

```diff
@@ -46,6 +49,7 @@ public void MefCtor_CheckIsExported()
         }
 
         [TestMethod]
+        [TestCategory("Integration")]
```

Comment:
Miscellaneous: added a category to help identify/exclude tests that are dependent on the environment.